File: weirentao/day4/main/java/com/example/demo/entity/Log.java
package com.example.demo.entity;


import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import org.springframework.format.annotation.DateTimeFormat;
import java.util.Date;
@Data
@TableName("mysyslog")
public class Log {
    @TableId(value="id",type = IdType.AUTO)
    private int id;
    @TableField("username")
    private String username;
    @TableField("operation")
    private String operation;
    @TableField("time")
    private long time;
    @TableField("hethod")
    private String hethod;
    @TableField("parms")
    private String parms;
    @TableField("ip")
    private String ip;
    @TableField("createtime")
    @DateTimeFormat(pattern="yyyy-MM-dd")
    private Date createtime;

}

File: caiguicheng/week01/day04/上课内容/demo04/entity/Log.kt
package com.example.demo04.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.annotation.TableName
import lombok.Data
import org.springframework.format.annotation.DateTimeFormat
import java.util.*

//@Data
@TableName("mysyslog")
class Log {
    @TableId(value = "id", type = IdType.AUTO)
    var id: Int = 0
    @TableField("username")
    var username: String = ""
    @TableField("operation")
    var operation: String = ""
    @TableField("time")
    var time: Long = 0L
    @TableField("method")
    var method: String = ""
    @TableField("params")
    var params: String = ""
    @TableField("ip")
    var ip: String = ""
    @TableField("createTime")
    @DateTimeFormat(pattern = "yyyy-MM-dd")
    var createTime: Date? = null
}

//@TableName("mysyslog")
//data class T(
//        @TableId(value = "id", type = IdType.AUTO)
//        var id: Int = 0,
//        @TableField("username")
//        var username: String = "",
//        @TableField("operation")
//        var operation: String = "",
//        @TableField("time")
//        var time: Long = 0L,
//        @TableField("method")
//        var method: String = "",
//        @TableField("parms")
//        var parms: String = "",
//        @TableField("ip")
//        var ip: String = "",
//        @TableField("createTime")
//        @DateTimeFormat(pattern = "yyyy-MM-dd")
//        var createTime: Date? = null
//)